src/Symfony/Component/RateLimiter/Policy/TokenBucketLimiter.php

```diff
@@ -9,11 +9,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\RateLimiter;
+namespace Symfony\Component\RateLimiter\Policy;
 
 use Symfony\Component\Lock\LockInterface;
 use Symfony\Component\Lock\NoLock;
 use Symfony\Component\RateLimiter\Exception\MaxWaitDurationExceededException;
+use Symfony\Component\RateLimiter\LimiterInterface;
+use Symfony\Component\RateLimiter\Rate;
+use Symfony\Component\RateLimiter\RateLimit;
+use Symfony\Component\RateLimiter\Reservation;
+use Symfony\Component\RateLimiter\ResetLimiterTrait;
```

Comment:
The `ResetLimiterTrait` should also be moved - it's used only by the policy classes.

Reply:
Sure. Will do